internal/utils/templates/init_config.toml

```diff
@@ -172,3 +172,4 @@ s3_region = "env(S3_REGION)"
 s3_access_key = "env(S3_ACCESS_KEY)"
 # Configures AWS_SECRET_ACCESS_KEY for S3 bucket
 s3_secret_key = "env(S3_SECRET_KEY)"
+functions_inspector_port = 8083
```

Comment:
would be good to have this under `edge-runtime` but also it's an experimental feature too. cc: @sweatybridge how do you handle flags when they move out of experimental?

Reply:
Configs in `[experimental]` section can be removed any time without backwards compatibility guarantee. That's the only difference.

I agree it's better to move `inspector_port` under `edge_runtime` section.

Reply:
(suggestion has been reflected to the PR)

cc @laktek @sweatybridge